Form/Type/CollapsibleCollectionType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sander
+ * Date: 08/04/15
+ * Time: 16:18
+ */
```

Comment:
Could you remove this part? You could add yourself as an author in the class comment